README.md

```diff
@@ -34,7 +29,45 @@ It has been developed during [GSoC 2018](https://jenkins.io/projects/gsoc/2018/c
 ## Release Notes
 See the [GitHub Releases](https://github.com/jenkinsci/code-coverage-api-plugin/releases).
 
-## How to Use it
+## Features
+### General Features
+* Pipeline support
+* Parallel execution in pipeline support
+* Reports combining
+* REST API
+* Failed conditions and flexible threshold setting
+
+### Coverage Analysis
```

Comment:
I think we can delete this part as well. Currently only pipeline support is working (and thresholds need to be changed as well).

Reply:
Yes, you are right - done. 
Do you already know when there will be the first version of the new parsers? ... since a rework of the thresholds would be really helpful as well. :)

Reply:
> Do you already know when there will be the first version of the new parsers? ... since a rework of the thresholds would be really helpful as well. :)

The new parsers have been merged already in https://github.com/uhafner/coverage-model.

Ihm currently integrating them in 
https://github.com/jenkinsci/code-coverage-api-plugin/pull/502

Let's see how far I get the implementation today. Then I will make a release of `coverage-model`.